LICENSE-binary

```diff
@@ -321,6 +321,7 @@ org.apache.kerby:kerby-pkix:1.0.1
 org.apache.kerby:kerby-util:1.0.1
 org.apache.kerby:kerby-xdr:1.0.1
 org.apache.kerby:token-provider:1.0.1
+org.apache.solr:solr-solrj:8.8.2
```

Comment:
I think this line can be removed because the solr jar file is not included the binary tarball.

Reply:
Hi Akira, it's actually bundled inside  the application catalog war file: hadoop-yarn-project/target/hadoop-yarn-project-3.4.0-SNAPSHOT/share/hadoop/yarn/hadoop-yarn-applications-catalog-webapp-3.4.0-SNAPSHOT.war

I am not familiar with this subproject... should we include dependencies from this war file?

Reply:
Thank you for the response.

```
% unzip -l ./hadoop-3.4.0-SNAPSHOT/share/hadoop/yarn/hadoop-yarn-applications-catalog-webapp-3.4.0-SNAPSHOT.war | grep solr
  2364784  11-19-2021 18:24   WEB-INF/lib/solr-solrj-8.8.2.jar
```

Given the jar file is actually in the binary tarball, we must add it in the LICENSE-binary file.